master.yaml

```diff
@@ -0,0 +1,34 @@
+---
+# mayaonline can parse this yaml to obtain tests specific to applications (based on tag) & populate test params,
+# fulfill pre-requisites etc, before deployment. The params are explicitly typed to prompt right values from user.
+# Test parameters can be of different kinds such as ENV, configmap etc., with their own properties.
+# '@userdata' is a placeholder which is either autofilled/userfilled. If empty, use defaults. The test has tags which
+# will help with categorization & application-mapping
+
+
+tests:
+  - name: pod_kill
+    test_tags: ['mysql', 'pumba']
```

Comment:
For generic tests should we put a predefined value (say `all`) in the `test_tags` like:
```
- name: pod_kill
  test_tags: ['all']
```

or should we have a seperate key-value to specify if the particular test is generic or not. like:
```
- name: pod_kill
  testType: 'generic' # or 'specific'
  ...
```